pkgs/by-name/cr/cronutils/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cronutils";
+  version = "1.10";
+
+  outputs = [ "out" "man" ];
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "cronutils";
+    rev = "version/${version}";
+    hash = "sha256-XJksfX4jqE32l4HipvO26iv9W4c0Iss6DenlEatdL1k=";
+  };
+
+  makeFlags = [ "DESTDIR=$(out)" "prefix=" ];
+
+  CFLAGS = lib.optional stdenv.isDarwin "-D_DARWIN_C_SOURCE";
```

```suggestion
  env.CFLAGS = lib.optionalString stdenv.isDarwin "-D_DARWIN_C_SOURCE";
```
Not sure if `env.NIX_CFLAGS_COMPILE` would be better here.